v2/rest/pulse.go

```diff
@@ -0,0 +1,136 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"path"
+	"strconv"
+
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/convert"
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/models/pulse"
+	"github.com/bitfinexcom/bitfinex-api-go/pkg/models/pulseprofile"
+	"github.com/bitfinexcom/bitfinex-api-go/v2"
+)
+
+type PulseService struct {
+	requestFactory
+	Synchronous
+}
+
+// PublicPulseProfile returns details for a specific Pulse profile
+// https://docs.bitfinex.com/reference#rest-public-pulse-profile
+func (ps *PulseService) PublicPulseProfile(nickname string) (*pulseprofile.PulseProfile, error) {
```

Comment:
Prehaps create a nickname type for this? Might help to explain the function a bit more?

```
type Nickname string
```